app/Http/Controllers/OrderController.php

```
@@ -87,11 +97,17 @@ public function store(StoreOrderRequest $request): JsonResponse
      * @param Order $order
      * @return JsonResponse
      */
-    public function show(Order $order): JsonResponse
+public function show(Order $order): JsonResponse
     {
-        return response()->json(
-            new OrderResource($order->load(['customer', 'assignedTo', 'createdBy', 'updatedBy', 'categories']))
-        );
+        $key = self::showKeyFor($order->uuid);
+        $hit = Cache::has($key);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'show'. (undefined)

(StaticAccess)